manager/Makefile

```diff
@@ -115,6 +118,7 @@ deploy_multi_it: manifests $(TOOLBIN)/kustomize $(TOOLBIN)/kubectl config/defaul
 	make build-ca-certs
 	make deploy-custom-ca-secret
 	$(TOOLBIN)/kubectl create namespace ${KUBE_NAMESPACE} || true
+	$(TOOLBIN)/kubectl create namespace m4d-blueprints || true
```

Comment:
Would it make sense to put this is a Namespace resource in the deployment? Otherwise you also require users to create this namespace

Reply:
I put the namespace creation into the deploy target as well. I think it should be created at the same place as the m4d-system namespace that's why I put it here. If we want to create the namespace as part of a yaml we should also do this for the m4d-system namespace. But I'd wait for that until the installation refactoring is done.

Reply:
I might be wrong but I don't think it's the same case as m4d-system (which is where you will install a chart to). This namespace is an installation side effect. No one would manually create it and neither should our scripts.